tests/tests.js

```diff
@@ -589,10 +589,8 @@ exports.defineAutoTests = function() {
                 removeContactsByFields(["note"], "DeleteMe", function() {
                     contact.save(function(c_obj) {
                         var findWin = function(cs) {
-                            // update to have proper saved id
-                            contact = cs[0];
                             expect(cs.length).toBe(1);
-                            done();
+                            done(cs[0]);
```

Comment:
@omefire @riknoll In case of specs 26 &27 - they are bound to different functions. But for specs 25, 28, 29 - they are bound to actual done functions (in which case the passed in parameter will just be ignored) 

But, the name need not be done. I agree. I changed it to callbacl and updated the PR.